deurdu/README.md

```diff
@@ -1,3 +1,22 @@
 This directory contains all data and source code for deurdu blogging
 website.
 
+#For frontend or client side - to make it activate
+1) First install node.js in system
+Goto --> 'https://nodejs.org/en/download/source-code'
+
+2) Make sure to have XAMPP CONTROL PANEL
```

Comment:
after installing `XAMPP` on Ubuntu 20.04, I see an new error about "Database not found"
```bash
nodemon index.js 
[nodemon] 3.1.7
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): *.*
[nodemon] watching extensions: js,mjs,cjs,json
[nodemon] starting `node index.js`
Server is running on port 3030
Database connection failed: Error: ER_BAD_DB_ERROR: Unknown database 'deurdu'
    at Sequence._packetToError (/home/q550356/sample_code/deurdu/server/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Handshake.ErrorPacket (/home/q550356/sample_code/deurdu/server/node_modules/mysql/lib/protocol/sequences/Handshake.js:123:18)
    at Protocol._parsePacket (/home/q550356/sample_code/deurdu/server/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/home/q550356/sample_code/deurdu/server/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/home/q550356/sample_code/deurdu/server/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/home/q550356/sample_code/deurdu/server/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/home/q550356/sample_code/deurdu/server/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/home/q550356/sample_code/deurdu/server/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (node:events:519:28)
    at addChunk (node:internal/streams/readable:559:12)
    --------------------
    at Protocol._enqueue (/home/q550356/sample_code/deurdu/server/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Protocol.handshake (/home/q550356/sample_code/deurdu/server/node_modules/mysql/lib/protocol/Protocol.js:51:23)
    at Connection.connect (/home/q550356/sample_code/deurdu/server/node_modules/mysql/lib/Connection.js:116:18)
    at file:///home/q550356/sample_code/deurdu/server/Controllers/blogControllers.js:15:4
    at ModuleJob.run (node:internal/modules/esm/module_job:234:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:473:24)
    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:123:5) {
  code: 'ER_BAD_DB_ERROR',
  errno: 1049,
  sqlMessage: "Unknown database 'deurdu'",
  sqlState: '42000',
  fatal: true
}
```

I see `mySQL` running

![xampp_running](https://github.com/user-attachments/assets/7edcbc1a-12bc-4b96-95cc-9fbce6e810d7)

Reply:
I had to create a database first, something like
```bash
$ ./mysql -h localhost -u root -p
Enter password: 
Welcome to the MariaDB monitor.  Commands end with ; or \g.
Your MariaDB connection id is 25
Server version: 10.4.32-MariaDB Source distribution

Copyright (c) 2000, 2018, Oracle, MariaDB Corporation Ab and others.

Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.

MariaDB [(none)]> CREATE DATABASE deurdu;
Query OK, 1 row affected (0,010 sec)

MariaDB [(none)]> 
```

And also set the username and password for the database. but afterwards it at least connects

```bash
$ nodemon index.js
[nodemon] 3.1.7
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): *.*
[nodemon] watching extensions: js,mjs,cjs,json
[nodemon] starting `node index.js`
Server is running on port 3030
Database connected
```

Reply:
This should be added to the `README`